ivy/functional/frontends/tensorflow/signal.py

```diff
@@ -38,6 +38,27 @@ def kaiser_window(window_length, beta=12.0, dtype=ivy.float32, name=None):
     return ivy.kaiser_window(window_length, periodic=False, beta=beta, dtype=dtype)
 
 
+# mdct
+@to_ivy_arrays_and_back
+def mdct(
+    signals,
+    frame_length,
+    window_fn=ivy.vorbis_window,
+    pad_end=False,
+    norm=None,
+    name=None,
+):
+    signals = ivy.asarray(signals)
+    return ivy.mdct(
```

Comment:
This function isn't implemented on the functional API
You'll need to implement a composition to it🤞

Reply:
Thanks for letting me know. It looks like I will need to implement the functions under experimental. Can you please confirm whether I need to implement the `mdct` function in the following files:
[ivy/data_classes/array/experimental/layers.py](https://github.com/unifyai/ivy/blob/main/ivy/data_classes/array/experimental/layers.py)
[ivy/data_classes/container/experimental/layers.py](https://github.com/unifyai/ivy/blob/main/ivy/data_classes/containera/experimental/layers.py)
[ivy/functional/container/experimental/layers.py](https://github.com/unifyai/ivy/blob/main/ivy/functional/ivy/experimental/layers.py)
[ivy/functional/ivy/experimental/layers.py](https://github.com/unifyai/ivy/blob/main/ivy/functional/ivy/experimental/layers.py)
[ivy/functional/backends/tensorflow/experimental/layers.py](https://github.com/unifyai/ivy/blob/main/ivy/functional/backends/tensorflow/layers.py)

Thank you.